doc/04-Getting-started.md

```diff
@@ -28,6 +28,67 @@ this is an advanced task you should not tackle at the early beginning.
 
 Take some time to really understand how to work with Icinga Director first.
 
+First steps on the Frontend
+====================================
+Create a host
+------------------
+
+First we create a `host template` which we will name `director-generic-host`. The default check we will use for a Host is `hostalive`.
+
+Click on the interface on Icinga Director > Host Objects > Add Host and `add a template`. For details have a look at the screenshots below:
+
+![add host section](screenshot/director/04-getting-started/01-add_a_host.png)
+
+![add host template](screenshot/director/04-getting-started/02-add_host_template.png)
+
+Now we can add our first host. Go to Icinga Director > Host Objects > Add Host and choose `Object`:
+
```

Comment:
Please highlight the imports field here and use fully qualified example domain names according RFC 2606. Display name could be a (short) host name.